lib/jekyll-admin/server/data.rb

```diff
@@ -4,16 +4,17 @@ class Server < Sinatra::Base
     EXTENSIONS = %w(yml json).freeze
 
     namespace "/data" do
-      get do
-        json DataFile.all.map(&:to_api)
-      end
-
-      get "/:path.?:ext?" do
+      get "/*?/?:path.?:ext?" do
```

Comment:
k.. so change frontend to send requests to `/data/foo-dir` or leave it at `/data/foo-dir/`?